app/controllers/api/v1/authentication_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Api::V1::AuthenticationController < ApiController
+  skip_before_action :authenticate_user!
+
+  def create
+    user = User.find_by(email: params[:email])
+    if user.valid_password? params[:password]
+      render json: { token: JsonWebToken.encode(sub: user.id) }
+    else
+      render json: { errors: ['Invalid email or password'] }
+    end
+  end
+end
```

Comment:
- [ ] Let us display the user-friendly message when a user created an account or registered as well as after signing in instead of displaying only token. 

example: `render json: @user, status: :created`

![user-5](https://user-images.githubusercontent.com/55635977/139906545-82256170-f1b2-4669-9ca0-8bec1492fcbb.png)